 ldap Pulling 
 ldap Error failed to resolve reference "docker.io/bitnami/openldap:2.6.0": docker.io/bitnami/openldap:2.6.0: not found
Error response from daemon: failed to resolve reference "docker.io/bitnami/openldap:2.6.0": docker.io/bitnami/openldap:2.6.0: not found
